### PR TITLE
You can now craft makeshift pistols

### PR DESCRIPTION
**Pull request by thelaughingbomb**
## About The Pull Request

Currently there is only one option when making a makeshift gun and thats the improvised shotgun, this PR adds a pistol that has a flat 5% chance to misfire in your hand, it shoots 9mm bullets (which are 20 damage) and has a fire rate of 1 (a third of stechkin). The gun holds 7 shots, only accepts being loaded 1 bullet at a time.

## Why It's Good For The Game

More ghetto guns

## Changelog
:cl:
add: Makeshift pistol
imageadd: Sprites for new pistol
/:cl:

### CODE REVIEW

**Comment by Ohagi-Chan** [comment]
Based

**Comment by zeskorion** [comment]
Rng gay

**Comment by ZombiesVsPlants** [comment]
more content is always a plus

**Comment by MCterra10** [comment]
Gun misfires are inherently rng, also go back to bee zesko and stop snooping on downstreams. It makes you seen jealous

File: austation/code/modules/projectiles/guns/ballistic/makeshift_pistol.dm [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+	fire_rate = 1 //one third of a stechkin
+	automatic = 0
+	weapon_weight = WEAPON_LIGHT
+
+
+/obj/item/gun/ballistic/automatic/pistol/aumakeshift/process_fire(atom/target, mob/living/user, message = TRUE, params = null, zone_override = "")
+	if(prob(5)) //I can hear zesko reeeeeing all ready
+		playsound(user, fire_sound, fire_sound_volume, vary_fire_sound)
+		user.visible_message("<span class='userdanger'>The [src] misfires and needs to be rechambered!</span>")
+		chambered = null //Basically it misfires and you gotta rechamber it
```
Comment (MCterra10) [inline austation/code/modules/projectiles/guns/ballistic/makeshift_pistol.dm:20]: Memory leak, please qdel the object in the chamber before nulling it out.

Comment (thelaughingbomb) [reply austation/code/modules/projectiles/guns/ballistic/makeshift_pistol.dm:21]: done

**Comment by MCterra10** [comment]
travis fail related to kangaroos.
should be fine

**Pull request merged**